libnet/sample/Makefile.am

```diff
@@ -77,3 +77,9 @@ test_ipv6_icmpv4_SOURCES = test_ipv6_icmpv4.c
 EXTRA_DIST = libnet_test.h
 
 LDADD 			= $(top_builddir)/src/libnet.la
+
+EXTRA_DIST += arp-new.c win32/arp/arp.vcproj win32/cdp/cdp.vcproj	\
```

Comment:
just "win32/" is fine to include folder + content
and I think append them to EXTRA_DIST like so:
`EXTRA_DIST = libnet_test.h arp-new.c win32/`